recipes/leptonica/all/conanfile.py

```diff
@@ -51,6 +51,8 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
+        if self.options.with_tiff:
+            self.options["libtiff"].jpeg = self.options.with_jpeg
```

Comment:
Is there a validate method?

It could be overridden by the user now AFAIK

Reply:
Do i understand it here correctly? Should the validation check for equality, to ensure that the user does not set a partial override?

Reply:
@cguentherTUChemnitz Yes. You could do:

```python
def validate(self):
    libtiff = self.dependencies["libtiff"]
    if libtiff.options.jpeg != self.info.options.with_jpeg:
        raise ConanInvalidConfiguration(f"{self.ref} requires option value {self.name}:with_jpeg equal to libtiff:jpeg.")
```